.azure-pipelines/1esmain.yml

```diff
@@ -48,7 +48,7 @@ extends:
       credscan:
         suppressionsFile: $(Build.SourcesDirectory)\.azure-pipelines\compliance\CredScanSuppressions.json
       componentgovernance:
-        ignoreDirectories: $(Build.SourcesDirectory)\**\.vscode-test
+        ignoreDirectories: $(Build.SourcesDirectory)\appservice\.vscode-test,$(Build.SourcesDirectory)\appsettings\.vscode-test,$(Build.SourcesDirectory)\auth\.vscode-test,$(Build.SourcesDirectory)\azure\.vscode-test,$(Build.SourcesDirectory)\dev\.vscode-test,$(Build.SourcesDirectory)\eslint_config_azuretools\.vscode-test,$(Build.SourcesDirectory)\github\.vscode-test,$(Build.SourcesDirectory)\serviceconnector\.vscode-test,$(Build.SourcesDirectory)\utils\.vscode-test
```

Comment:
`eslint_config_azuretools` were these meant to be underscores?

Reply:
No, great catch.